addons/popochiu/engine/others/popochiu_utils.gd

```diff
@@ -3,6 +3,11 @@ class_name PopochiuUtils
 extends Node
 ## Utility functions for Popochiu.
 
+## Used for setting the double click delay. Windows default is 500 miliseconds.
+static var double_click_delay: float = 0.35 # 0.5 felt like too long of a delay before acting
+## Used for tracking if a double click has occured.
+static var has_double_click: bool = false
```

Comment:
Just to mention, in this case, because the types are `float` and `bool` there is no need to specify those in the variable declaration. You can just simply do: `static var double_click_delay := 0.35` and Godot will know it is a float. Same for the `bool` variable.

Reply:
My understanding is if you used the types on the variables then it improves the auto code completion and can also help with godot giving you warnings or errors on your code before running the game.

The documentation also suggests fully using typed or dynamic code for consistency.

https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/static_typing.html

If you want to have functions typed and variable declarations not typed then that is easy enough to do. I am wondering how that effects code completion in particular. I have not used godot enough to know the difference in code completion quality particularly as I tend to add the types on the little things I have done so far.

Reply:
I see what you mean.

var f1 = 1.0 # dynamic
var f2 := 1.0 # statically typed with inference